pkg/cmd/run/shared/shared.go

```diff
@@ -42,6 +43,19 @@ type Status string
 type Conclusion string
 type Level string
 
+var RunFields = []string{
+	"name",
+	"commitMsg",
```

Comment:
I wouldn't expose the `commitMsg` function:
1. It's basically a view helper for our own logic
2. It does not allow access to the full commit message

It might be much better to just expose the raw `head_commit` object? Or, if we expose `head_sha`, we can omit `head_commit` and `commitMsg` altogether.